.github/workflows/test.yml

```diff
@@ -0,0 +1,15 @@
+name: CI
+
+on: pull_request
+
+concurrency:
+  group: ${{ github.ref_name }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    name: Tests
+    uses: adamruzicka/actions/.github/workflows/test-gem.yml@test-gem-packages
+    with:
+      command: bundle exec rake test
+      extra_packages: libopenscap8
```

Comment:
On Ubuntu 24.04 there's [openscap-scanner](https://packages.ubuntu.com/noble/openscap-scanner) which provides `/usr/bin/oscap` while `libopenscap8` no longer exists on Ubuntu 24.04. I'm surprised this still runs on Ubuntu 22.04 because in some of our jobs we've started to see `ubuntu-latest` being 24.04. We can expect to see failures around this when upgrading.

Reply:
I originally used openscap-scanner, but then found out the test did run on jammy, which doesn't have it.  So yeah, I'm aware that it might fail eventually, I'll deal with that once it happens.